Gordon360/Models/ViewModels/RecIM/ParticipantExtendedViewModel.cs

```diff
@@ -32,6 +32,8 @@ public class ParticipantExtendedViewModel
 
         public static implicit operator ParticipantExtendedViewModel(ParticipantView p)
```

Comment:
You should add nullability to these types
```suggestion
        public static implicit operator ParticipantExtendedViewModel?(ParticipantView? p)
```